lib/config.js

```diff
@@ -883,7 +884,8 @@ util.parseFile = function(fullFilename) {
       configObject = require(fullFilename);
     }
     else if (extension === 'iced') {
-      Iced = require("iced-coffee-script");
+      var dep = "iced-coffee-script"
```

Comment:
Javascript hoists variables, and having individual `var dep` gives the misleading impression that these are different things, when they are all the same variable.

A better pattern would be to define `var dep = undefined` at the top of the function and use that variable without (misleadingly) re-defining it in each instance.

Even better would be to have a block at the top of the file like this

```
// Define soft dependencies so transpilers don't include everything
var ICED_COFFEE_SCRIPT = "iced-coffee-script";
var JS_YAML = "js-yaml"
...
```
Then simply use these variables instead of the hardcoded strings `require(ICED_COFFEE_SCRIPT);`